src/tools/ResultsComparer/ResultsComparer.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFrameworks>$(PYTHON_SCRIPT_TARGET_FRAMEWORKS)</TargetFrameworks>
+    <TargetFrameworks Condition="'$(TargetFrameworks)' == ''">netcoreapp2.1;netcoreapp3.0;netcoreapp5.0</TargetFrameworks>
```

Comment:
I wonder if in this case we should go the other way - leave it as 2.1 in the case that it isn't set. That will work fine on almost anyone's machine, and in the lab we'll have `netcoreapp5.0` where we need it. (`Microbenchmarks.csproj` is more of a special case because we expect it to only run with latest CLIs. Other tools can be more relaxed.)

Reply:
Smart.